040-SDK/135-sql-access.mdx

```diff
@@ -148,37 +148,44 @@ Response:
 <TabbedCode tabs={['TypeScript', 'Python', 'JSON']}>
 ```ts
 const user = { name: "Keanu Reeves", email: "keanu@example.com" };
-const { records: [record] } = await xata.sql<TeamsRecord>`INSERT INTO "Users" (name, email) VALUES (${user.name}, ${user.email})`;
```

Comment:
By default we should suggest this syntax instead of the object one. It's safer (built-in SQL injection protection) and easier to understand.